cool_seq_tool/data_sources/uta_database.py

```diff
@@ -4,9 +4,9 @@
 import logging
 from typing import Dict, List, Optional, Tuple, Any, TypeVar, Type, Union
 from os import environ
-from urllib.parse import quote, unquote
+from urllib.parse import quote, unquote, urlparse, ParseResult as UrlLibParseResult
 
-from six.moves.urllib import parse as urlparse
+# from six.moves.urllib import parse as urlparse
```

Comment:
```suggestion
```

Reply:
![Screenshot 2023-08-04 at 3 36 29 PM](https://github.com/GenomicMedLab/cool-seq-tool/assets/12942397/40056bb3-a11a-484b-a438-4b94d8a4d997)

that was close